app/src/main/java/app/revanced/manager/ui/screen/settings/ContributorScreen.kt

```diff
@@ -50,87 +54,139 @@ fun ContributorScreen(
                 .fillMaxHeight()
                 .padding(paddingValues)
                 .fillMaxWidth()
-                .verticalScroll(rememberScrollState())
+                .verticalScroll(rememberScrollState()),
+            verticalArrangement = if (repositories.isNullOrEmpty()) Arrangement.Center else Arrangement.Top
         ) {
-            if(repositories.isEmpty()) {
-                LoadingIndicator()
-            }
-            repositories.forEach {
-                ExpandableListCard(
-                    title = it.name,
-                    contributors = it.contributors
-                )
-            }
+            repositories?.let {
+                if (repositories.isEmpty()) {
+                    Text(
+                        text = stringResource(id = R.string.no_contributors_found),
+                        style = MaterialTheme.typography.titleLarge
+                    )
+                } else {
+                    Column(
+                        modifier = Modifier.padding(16.dp),
+                        verticalArrangement = Arrangement.spacedBy(24.dp)
+                    ) {
+                        repositories.forEach {
+                            ContributorsCard(
+                                title = it.name,
+                                contributors = it.contributors
+                            )
+                        }
+                    }
+                }
+            } ?: LoadingIndicator()
```

Comment:
A `LazyColumn` would probably be better here considering the time complexity of the UI code

Reply:
How can I keep the padding of that Column in the middle? applying it to the parent LazyColumn does this
![image](https://github.com/ReVanced/revanced-manager/assets/79272171/5e4ad8ae-0924-4a67-8764-02a7c4f09a7e)

Reply:
Turning the `Column` that contains the `ContributorsCard`s into a `LazyColumn` should work then. The parent's `verticalScroll` would have to be removed though.

Reply:
That didn't work, it's because the LazyColumn is the part that's scrolling, instead of the whole page

Reply:
I'll look into this later today to see if there is a way to use `LazyColumn`.

Reply:
It seems I will be kinda busy this weekend. Can you do a basic performance test on a release build when you have time?

We can keep it as a regular column if the performance is fine.

Reply:
I have no experience with profiling apps, but following https://developer.android.com/studio/profile I have these results:

Column:
- CPU: 0% to 29%
- Memory: 49MB to 61MB

![image](https://github.com/ReVanced/revanced-manager/assets/79272171/46331d42-cbe3-432e-9886-b88df4a15703)


LazyColumn:
- CPU: 0% to 18%
- Memory: 49MB to 52MB

![image](https://github.com/ReVanced/revanced-manager/assets/79272171/9e086233-5c32-4cb5-b893-0b541c2550a5)


Although the results I've gotten were not consistent in each run, it consistently had a reasonable difference between LazyColumn and Column

Reply:
Turns out, LazyColumn has a "contentPadding" parameter that acts the same as padding in web does, so this is solved